_episodes/02-filedir.md

```diff
@@ -442,7 +442,7 @@ To navigate through the `man` pages,
 you may use the up and down arrow keys to move line-by-line,
 or try the "b" and spacebar keys to skip up and down by a full page.
 To search for a character or word in the `man` pages, 
-use "/" followed by the character or word you are searching for. 
+use "/" followed by the character or word you are searching for. At times, a search will result in multiple hits.  If so, you can move between hits using `n` key (for moving forward) and `N` key (for moving backward).
```

Comment:
1. The surrounding text uses double quotes when referring to a key, e.g., "b", not `b`. What convention to use?
2. It may be confusing to refer to both an "n" key and also an "N" key (there is only one N key on the keyboard). Perhaps we should avoid use of the word "key" throughout. How does this sound?

> At times, a search will result in multiple hits.  If so, you can move between hits using `n` (for moving forward) and `N` (for moving backward